### PR TITLE
Console proxy

### PR DESCRIPTION
**Pull request by ntartania**
I implemented the separate wifiAntenna as discussed this afternoon, merged and fixed the conflicts, the "compare" view looks really clean, it seems to work from my perspective... but it can't be merged automatically, I think there may be an issue with the repeated commits in the history.

Please review @cruizm @pgavigan @gtrabes and we can see how/if to merge.

### CODE REVIEW

File: saviSimulation/savi/jason_processing/UAS.java [hunk 1]
```diff
@@ -89,6 +96,28 @@ public void draw() {
 		
 		
 	}
-	
+
+	@Override
+	public  List<String> getOutgoingMessages(){
+		
+		List<String> myMsgOutCopy = new LinkedList<String>();
+		myMsgOutCopy.addAll(getBehavior().agentState.getMsgOutAll());
```
Comment (pgavigan) [inline saviSimulation/savi/jason_processing/UAS.java]: This line feels a little convoluted, but I think I understand why. I see that the linkage to the agentState lives in the behaviour. Is there any way we can simplify this?

**Comment by cruizm** [comment]
Hello,

I reviewing the PR, and I get many conflicts. I think they are not trivial.
I am trying to solve them with Git Kraken but I am facing issues. 
I will continue looking at it.

**Comment by pgavigan** [comment]
To clarify, my approval of the PR is that I like the state of the software in this branch. Definitely, the conflicts will need to be addressed and resolved in order to merge.

**Comment by ntartania** [comment]
fixed conflicts manually.

**Pull request merged**